lib/vagrant-google/action/read_ssh_info.rb

```diff
@@ -42,11 +42,24 @@ def read_ssh_info(google, machine)
             return nil
           end
 
-          # Read SSH network info
-          return {
+          # Get private_ip setting
+          use_private_ip = machine.provider_config.get_zone_config(zone).use_private_ip
+
+          # Default to use public ip address
+          ssh_info = {
             :host => server.public_ip_address,
             :port => 22
           }
+
+          if use_private_ip then
```

Comment:
@utdrmac No prob, I'll fix in the release patch.
Just an FYI: https://github.com/bbatsov/ruby-style-guide#no-then